pandas/tests/interchange/test_impl.py

```diff
@@ -410,7 +407,6 @@ def test_empty_string_column():
     tm.assert_frame_equal(df, result)
 
 
-@pytest.mark.xfail(using_string_dtype(), reason="TODO(infer_string)")
 def test_large_string():
     # GH#56702
     pytest.importorskip("pyarrow")
```

Comment:
Just below here (can't comment on the line), we still hardcode to object dtype, that should be 

```
expected = pd.DataFrame({"a": ["x"]}, dtype="str")
```